File: backend c#/Marketplace/UserService/Marketplace/DTO/UserDTO.cs
namespace UserService.DTO
{
    public class UserDTO
    {
        public int Id { get; private set; }

        public string Login { get; set; } = null!;

        public string Password { get; set; } = null!;

        public string Email { get; set; } = null!;

        public string Role { get; set; } = null!;

    }
}

File: backend c#/Marketplace/SMTPAlert/src/main/java/server/lost/SMTPAlert/UserServices/UserDTO.java
package server.lost.SMTPAlert.UserServices;

public class UserDTO {
    private String id;
    private String login;
    private String password;
    private String email;
    private String role;

    // Геттеры и сеттеры

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

}
